src/cmd-koji-upload

```diff
@@ -470,7 +474,10 @@ class Upload():
                 "owner": self._owner,
                 "source": source['origin'],
                 "start_time": stamp,
-                "version": f"{self.build.build_id}"
+                # RHCOS wants to be semver-compatible, but Koji doesn't
+                # accept `-`.  See
+                # https://github.com/openshift/oc/pull/209#issuecomment-564876535
+                "version": f"{self.build.build_id.replace('-', '.')}"
```

Comment:
Needed to fold in this PR versus keep in https://github.com/coreos/coreos-assembler/pull/1053

Reply:
I wanted to _test_ the code path. It was easier to include this commit in the pipeline for testing.